SofaKernel/modules/SofaBaseCollision/DiscreteIntersection.cpp

```diff
@@ -95,5 +86,13 @@ ElementIntersector* DiscreteIntersection::findIntersector(core::CollisionModel*
 
 } // namespace component
 
+namespace core
+{
+namespace collision
+{
+template class SOFA_BASE_COLLISION_API IntersectorFactory<component::collision::DiscreteIntersection>;
+}
+}
+
```

Comment:
I think you had a bad merge here. Could you double check with the current master branch but I think this is reverting a fix.